src/lib/workspace/AllAnalyses.tsx

```diff
@@ -78,6 +78,7 @@ const useStyles = makeStyles({
 });
 
 const UNKNOWN_DATASET_NAME = 'Unknown study';
+export const ANALYSIS_NAME_MAX_LENGTH = 50;
```

Comment:
This `ANALYSIS_NAME_MAX_LENGTH` constant should probably go into a utility file - maybe [here](https://github.com/VEuPathDB/web-eda/blob/main/src/lib/core/utils/analysis.ts)?